RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+Follow the steps outlined in this document to release a new version of `cloudgov-style`.
+
+### Blah blah
```

Comment:
I think we should describe the point of releasing and we're we'll be publishing to. Its kind of a different sort of release process that tagging this also publishes an npm module and a ruby gem, so we should make that explicit.